H/HelFEM/common.jl

```diff
@@ -64,9 +62,9 @@ products = [
 dependencies = [
     BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
     Dependency("libcxxwrap_julia_jll"),
-    Dependency(PackageSpec(name = "armadillo_jll", version = "9.850.1")),
-    Dependency(PackageSpec(name = "GSL_jll", version = "2.6.0")),
-    Dependency(PackageSpec(name = "OpenBLAS_jll", version = "0.3.9")),
+    Dependency("armadillo_jll", compat = "9.850.1"),
+    Dependency("GSL_jll", compat = "2.6.0"),
+    Dependency("OpenBLAS_jll", compat = "0.3.13"),
```

Comment:
OpenBLAS v0.3.13 is in Julia v1.7, so this is not really going to work.  OpenBLAS v0.3.10 is what is used in Julia v1.6 and it still doesn't do ILP64 for aarch64